init-script

```diff
@@ -176,45 +228,45 @@ inject_loop() {
     mkfifo $INJ_STDIN
     echo "This entire directory is for debugging init - it can safely be removed" > $INJ_DIR/README
 
-    echo "########################## Beginning inject loop"
+    log "########################## Beginning inject loop"
     while : ; do
         while read IN; do
-	    if [ "$IN" = "continue" ]; then break 2;fi
```

Comment:
Please remove code style changes.. I am not fan of them.. they break the git blame/git log functionality..